src/ReactionWheel.lf

```diff
@@ -298,7 +294,7 @@ reactor SimulatorInterface {
 }
 
 main reactor {
-    sim = new MujocoSimulator(file="../3_axis_reaction_wheel.xml");
+    sim = new MujocoSimulator(file="models/3_axis_reaction_wheel.xml");
```

Comment:
This assumes that at the location where `./ReactionWheel` is invoked. There needs to be a `models` directory. For raspberry pi, we first copy the `src-gen` to the board and then make it locally. This change means that we need to create a `models` directory with `3_axis_reaction_wheel.xml` where we call `ReactionWheel`. Is there a way to satisfy  this requirement?

Reply:
Also note there is this workflow: https://www.lf-lang.org/docs/developer/on-target-development

Reply:
I changed it because it made sense from the LF project organization perspective. Where you have `bin/ReactionWheel` and `models/....xml`. I didnt look into how the cross-compilation and deployment on the RPi was done, probably a good idea if it compiles and runs on both platforms. 

You could copy the entire project onto the RPi4 and move the binary to the `bin` directory also on the RPi and invoke it from the root.

Or we just use the `../` path and you have to execute the binary from `src-gen/ReactionWheel/build` for it to run on the host machine also

Reply:
@erlingrj  I agree that it's important to make development as streamlined as possible. Let's adopt your proposal! I can adjust the shell script to copy the src-gen inside the project folder.